cli/js/headers_test.ts

```diff
@@ -329,3 +330,25 @@ test(function toStringShouldBeWebCompatibility(): void {
   const headers = new Headers();
   assertEquals(headers.toString(), "[object Headers]");
 });
+
+function stringify(...args: unknown[]): string {
+  return stringifyArgs(args).replace(/\n$/, "");
+}
+
+test(function customInspectReturnsCorrectHeadersFormat(): void {
+  const blankHeaders = new Headers();
+  assertEquals(stringify(blankHeaders), "Headers {}");
+  const singleHeader = new Headers([["Content-Type", "application/json"]]);
+  assertEquals(
+    stringify(singleHeader),
+    "Headers {content-type: application/json}"
+  );
+  const multiParamHeader = new Headers([
+    ["Content-Type", "application/json"],
+    ["Content-Length", "1337"]
+  ]);
+  assertEquals(
+    stringify(multiParamHeader),
+    "Headers {content-type: Application/json, content-length: 1337}"
```

Comment:
This should have spaces near the braces: `Headers { content-type: ...` for consistency with how objects are printed.